src/phasorpy/phasor.py

```diff
@@ -282,47 +482,154 @@ def phasor_to_polar(
     return phase, modulation
 
 
+def phasor_from_polar(
+    phase: ArrayLike,
+    modulation: ArrayLike,
+    /,
+) -> tuple[NDArray[Any], NDArray[Any]]:
+    """Return phasor coordinates from polar coordinates.
+
+    Parameters
+    ----------
+    phase : array_like
+        Angular component of polar coordinates in radians.
+    modulation : array_like
+        Radial component of polar coordinates.
+
+    Returns
+    -------
+    real : ndarray
+        Real component of phasor coordinates.
+    imag : ndarray
+        Imaginary component of phasor coordinates.
+
+    Raises
+    ------
+    ValueError
+        The shapes of `phase` and `modulation` do not match.
+
+    Examples
+    --------
+    Calculate phasor coordinates from three polar coordinates:
+
+    >>> phasor_from_polar(
+    ...     [0.0, math.pi / 4, math.pi / 2], [1.0, math.sqrt(0.5), 1.0]
+    ... )  # doctest: +NUMBER
+    (array([1, 0.5, 0.0]), array([0, 0.5, 1]))
+
+    """
+    phase = numpy.asarray(phase)
+    modulation = numpy.asarray(modulation)
+    if phase.shape != modulation.shape:
+        raise ValueError(f'{phase.shape=} != {modulation.shape=}')
+    real = numpy.cos(phase)
+    real *= modulation
+    imag = numpy.sin(phase)
+    imag *= modulation
+    return real, imag
+
+
 def phasor_from_lifetime(
-    frequency: float,
+    frequency: ArrayLike,
     lifetime: ArrayLike,
     /,
     fraction: ArrayLike | None = None,
     *,
-    is_preexp: bool = False,
+    preexponential: bool = False,
+    unit_conversion: float = 1e-3,
+    squeeze: bool = True,
 ) -> tuple[NDArray[numpy.float64], NDArray[numpy.float64]]:
```

Comment:
How about allowing frequency and lifetime to be named arguments? That could make the tutorial clearer.
```Python
def phasor_from_lifetime(
    frequency: ArrayLike,
    lifetime: ArrayLike,
    fraction: ArrayLike | None = None,
    *,
    is_preexp: bool = False,
) ->
```

Reply:
Yes it can look more clear and easy to understand I think.

Reply:
Done